File: src/main/java/org/lanternpowered/server/text/selector/LanternSelectorType.java
package org.lanternpowered.server.text.selector;

import org.lanternpowered.server.catalog.SimpleLanternCatalogType;
import org.spongepowered.api.text.selector.SelectorType;
import org.spongepowered.api.util.annotation.NonnullByDefault;

@NonnullByDefault
public class LanternSelectorType extends SimpleLanternCatalogType implements SelectorType {

    public LanternSelectorType(String identifier) {
        super(identifier);
    }
}

File: src/main/kotlin/org/lanternpowered/server/text/selector/LanternSelectorType.kt
/*
 * Lantern
 *
 * Copyright (c) LanternPowered <https://www.lanternpowered.org>
 * Copyright (c) SpongePowered <https://www.spongepowered.org>
 * Copyright (c) contributors
 *
 * This work is licensed under the terms of the MIT License (MIT). For
 * a copy, see 'LICENSE.txt' or <https://opensource.org/licenses/MIT>.
 */
package org.lanternpowered.server.text.selector

import org.lanternpowered.api.ResourceKey
import org.lanternpowered.server.catalog.DefaultCatalogType
import org.spongepowered.api.text.selector.SelectorType

class LanternSelectorType(key: ResourceKey, val code: String) : DefaultCatalogType(key), SelectorType
